assets/js/components/notifications/SubtleNotification.stories.js

```diff
@@ -92,146 +90,6 @@ SuccessWithoutIcon.args = {
 };
 SuccessWithoutIcon.scenario = {};
 
-export const Warning = Template.bind( {} );
-Warning.storyName = 'Warning';
-Warning.args = {
-	title: 'Warning! Your Conversion ID was not added to your site',
-	dismissLabel: 'Ok, got it',
-	variant: 'warning',
-};
-Warning.scenario = {};
```

Comment:
Could we please delete ALL the relevant `google-site-kit_Components_SubtleNotification_[Info*|Warning]*` VRT scenario reference images as well?

Reply:
This has been addressed, let's wait for the VRT test to run to see if this is okay.